### DISCUSSION TITLE
How can I custom validation URL (ACTION_URL)

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#2097] [Q&A] [answered]
Can I custom the validation URL ? 

`mysite.com/_/.....` ==> my custom url/.....

My purpose I want after validated, it will redirect to /login page (my front end) and login

**Comment by ganigeorgiev** [comment, marked as answer]
You can change the default email template actiuon url from the:
Admin UI > Settings > Mail Settings > Default "Verification" email template > Edit the "Action URL" input.

**Reply by hungcrush** [reply]
I'm sorry. I saw that... Thank you
